NitroxClient/GameLogic/InitialSync/EscapePodInitialSyncProcessor.cs

```diff
@@ -15,7 +15,7 @@ public EscapePodInitialSyncProcessor(EscapePodManager escapePodManager)
 
         public override void Process(InitialPlayerSync packet)
         {
-            EscapePodModel escapePod = packet.EscapePodsData.Find(x => x.Id == packet.AssignedEscapePodId);
```

Comment:
I would recommend leaving as "==" considering that the "==" operator was added

Reply:
Yes, forgot to change that back, should i close this pr adn resubmit with the changes?